package.json

```diff
@@ -49,7 +49,7 @@
     "@electron-forge/maker-zip": "^6.2.1",
     "@electron-forge/plugin-auto-unpack-natives": "^6.2.1",
     "babel-loader": "^9.1.2",
-    "babel-plugin-macros": "^2.6.1",
+    "babel-plugin-macros": "^2.4.2",
```

Comment:
Why the downgrade?

Reply:
It caused a weird conflict with another peer dependency - I think it was `fs` and `fs-extra`?